modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -1106,17 +1162,10 @@ private <T> void _testFilterByRelatedObjectDefinitionSystemObjectField(
 			String systemPropertyName)
 		throws Exception {
 
-		String filter = _createFilter(
-			filterOperator,
-			_buildRelationshipsPropertyNameSyntax(
-				objectRelationship, systemPropertyName),
-			relatedObjectEntryValue);
-
-		String endpoint = StringBundler.concat(
-			objectDefinition.getRESTContextPath(), "?filter=", filter);
-
-		_testFilterObjectEntriesByRelatedObjectEntriesUsingAFilterOperator(
-			endpoint, expectedObjectFieldName, expectedObjectFieldValue);
+		_testFilterWithComposeFiltersByRelatedObjectDefinitionSystemObjectField(
```

Comment:
I don't know if the first method should exist as you are just invoking another method:

```
	private <T> void _testFilterByRelatedObjectDefinitionSystemObjectField(
			String expectedObjectFieldName, T expectedObjectFieldValue,
			FilterURLCreatorUtil.FilterOperator filterOperator,
			String leftFilter, ObjectDefinition objectDefinition,
			ObjectRelationship objectRelationship, T relatedObjectEntryValue,
			String rightFilter, String systemPropertyName)
		throws Exception {

		_testFilterObjectEntriesByRelatedObjectEntriesUsingAFilterOperator(
			expectedObjectFieldName, expectedObjectFieldValue, filterOperator,
			_buildRelationshipsPropertyNameSyntax(
				objectRelationship, systemPropertyName),
			leftFilter, objectDefinition, relatedObjectEntryValue, rightFilter);
	}

	private <T> void
			_testFilterObjectEntriesByRelatedObjectEntriesUsingAFilterOperator(
				String endpoint, String expectedObjectFieldName,
				T expectedObjectFieldValue)
		throws Exception {

		JSONObject jsonObject = HTTPTestUtil.invoke(
			null, endpoint, Http.Method.GET);

		JSONArray itemsJSONArray = jsonObject.getJSONArray("items");

		Assert.assertEquals(1, itemsJSONArray.length());

		JSONObject itemJSONObject = itemsJSONArray.getJSONObject(0);

		Assert.assertEquals(
			expectedObjectFieldValue,
			itemJSONObject.getInt(expectedObjectFieldName));
	}
```